### DISCUSSION TITLE
Pdf Summarizer project

### DISCUSSION THREAD

**Discussion by PiotrWarzachowski** [janhq/jan#1396] [General]
I wanted to contribute a bit by making this open source project, where **I used a ChatGPT to summarize PDF documents**, you can **contribute to change it to jan** if you want, here is my repo : https://github.com/PiotrWarzachowski/DocMind Feel free to check it out, **its public on vercel**. You can leave a star ⭐ if you want 💓. Have a good year

**Comment by PiotrWarzachowski** [comment]
I want to bump it to make as many people as possible see it 💓

**Comment by TomLucidor** [comment]
If it is not designed to be mostly local-AI then this vibes more like an Ad
